owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -56,10 +56,36 @@ class ReleaseNotesViewModel(
                 type = ReleaseNoteType.ENHANCEMENT,
             ),
             ReleaseNote(
-                title = R.string.release_notes_4_3_0_title_3,
-                subtitle = R.string.release_notes_4_3_0_subtitle_3,
+                title = R.string.release_notes_4_3_0_title_new_ui_manage_accounts,
+                subtitle = R.string.release_notes_4_3_0_subtitle_new_ui_manage_accounts,
+                type = ReleaseNoteType.ENHANCEMENT
+            ),
+            ReleaseNote(
+                title = R.string.release_notes_4_3_0_title_manual_removal_local_storage,
+                subtitle = R.string.release_notes_4_3_0_subtitle_manual_removal_local_storage,
                 type = ReleaseNoteType.ENHANCEMENT,
             ),
+            ReleaseNote(
+                title = R.string.release_notes_4_3_0_title_password_generator,
+                subtitle = R.string.release_notes_4_3_0_subtitle_password_generator,
+                type = ReleaseNoteType.ENHANCEMENT
+            ),
+
+            ReleaseNote(
+                title = R.string.release_notes_4_3_0_title_create_open_shortcut,
+                subtitle = R.string.release_notes_4_3_0_subtitle_create_open_shortcut,
+                type = ReleaseNoteType.ENHANCEMENT
+            ),
```

Comment:
This one is a change that they can find easily as well, since it is in the main FAB